addons/l10n_eu_oss/models/eu_field_map.py

```diff
@@ -0,0 +1,102 @@
+"""
+The EU_FIELD_MAP contains localization specific fields for account.tax
+"""
+EU_FIELD_MAP = {
+    # Austria
+    'at': {
+    },
+    # Belgium
+    'be_comp': {
+
+    },
+    'be_asso': {
+    },
+    # Bulgaria
+    'bg': {
+    },
+    # Croatia
+    'hr': {
+    },
+    # Cyprus
+    'cy': {
+    },
+    # Czech - Done in 13.0 - CoA not available yet
+    'cz': {
+    },
+    # Denmark
+    'dk': {
+    },
+    # Estonia - Done in 13.0 - CoA not available yet
+    'ee': {
+    },
+    # Finland
+    'fi': {
+    },
+    # France
+    'fr': {
+    },
+    # Germany SKR03
+    'de_skr03': {
+    },
+    # Germany SKR04
+    'de_skr04': {
+    },
+    # Greece
+    'gr': {
+    },
+    # Hungary
+    'hu': {
+    },
+    # Ireland
+    'ie': {
+    },
+    # Italy
+    'it': {
+    },
+    # Latvia
+    'lv': {
+    },
+    # Lithuania
+    'lt': {
+    },
+    # Luxembourg
+    'lu': {
+    },
+    # Malta - Done in 13.0 - CoA not available yet
+    'mt': {
+    },
+    # Netherlands
+    'nl': {
+    },
+    # Poland
+    'pl': {
+    },
+    # Portugal
+    'pt': {
+    },
+    # Romania
+    'ro': {
+    },
+    # Slovakia - Done in 13.0 - CoA not available yet
+    'sk': {
+    },
+    # Slovenia
+    'si': {
+    },
```

Comment:
I think we can skip all these countries/templates for now...
we can always add them later if need be.